File: Java/프로그래머스/0/120835. 진료 순서 정하기/진료 순서 정하기.java
import java.util.*;
class Solution {
    public int[] solution(int[] emergency) {
        int[] answer = new int[emergency.length];
        int[] array= new int[emergency.length];
        for(int i=0; i< emergency.length;i++){
            array[i]=emergency[i];
        }
        Arrays.sort(emergency); // emergency정렬
        int[] turn = new int[emergency.length];
        int n=emergency.length;
        for(int i=0; i<emergency.length;i++){
            turn[i]=n;
            n--;
        }
        int sameindex=0;
        int k=0;
        for(int i=0; i<emergency.length ; i++){
            for(int j=0; j<emergency.length;j++){
                if(emergency[i]==array[j]) {
                    sameindex=j;
                }
            }
            answer[k]=turn[sameindex];
            k++;
        }
        return answer;
    }
}

File: 프로그래머스/0/120835. 진료 순서 정하기/진료 순서 정하기.js
function solution(emergency) {
    var answer = new Array(emergency.length);
    var array= new Array(emergency.length);
    for(let i=0; i< emergency.length;i++){
        array[i]=emergency[i];
    }
    emergency.sort((a,b) => a-b); // emergency정렬
    let turn = new Array(emergency.length);
    let n=emergency.length;
    for(let i=0; i<emergency.length;i++){
        turn[i]=n;
        n--;
    }
    let sameindex=0;
    let k=0;
    for(let i=0; i<emergency.length ; i++){
        for(let j=0; j<emergency.length;j++){
            if(emergency[i]==array[j]) {
                sameindex=j;
            }
        }
        answer[k]=turn[sameindex];
        k++;
    }
    return answer;
}